templates/serverpod_templates/modulename_server/test/integration/example_endpoint_test.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:test/test.dart';
+
+// Import the generated file, it contains everything you need.
+import 'test_tools/serverpod_test_tools.dart';
+
+void main() {
+  withServerpod('Given Example endpoint', (sessionBuilder, endpoints) {
+    test('when calling `hello` then should return greeting', () async {
```

Comment:
Suggestion: Change to `when calling "hello" with name then returned greeting includes name`.

Here we are setting the standard for how tests are written. Could be good to spend some time to make sure this is forumlated in a good way :)

Reply:
Absolutely 👍

Reply:
[0b363f1](https://github.com/serverpod/serverpod/pull/2882/commits/0b363f114f37af63e6437992ca3f7066f2904259)